contexts/chat.tsx

```diff
@@ -70,9 +70,15 @@ interface ChatContextState {
   interrupt: () => void;
   fetchThreads: () => void;
   restartScript: () => void;
+  switchToThread: (
+    script: string,
+    id: string,
+    scriptId: string
+  ) => Promise<void>;
 }
 
 const defaultScriptName = `Tildy`;
+const notFoundScriptName = `[Deleted Assistant]`;
```

Comment:
This name seems to be used when a script is not found for some reason. Using `[Deleted Assistant]` seems like an assumption. Can we using something like `[Assistant Not Found]`?

Reply:
Yeah I kept going back and forth between the 2. Hearing it from someone else, I have to agree